lib/loader/index.js

```diff
@@ -47,6 +57,27 @@ function validateArguments (url, options, callback) {
 
 module.exports = {
   /**
+   * Clears the loaded URLs
+   *
+   * Clears any URLs that might have been stored in the loadedUrls hash, either
+   * locally or in the given namespace(optional). This is needed for testing so
```

Comment:
That is correct. During testing, I'm able to access namespace.loadedUrls but not the internal loadedUrls because I'm not exporting it. The best way I've thought of doing it is with this function, but as you said if it's only purpose is for testing, it would be nice to remove it.

I don't think there's a need for it in actual execution, but I've not had any luck removing it and still being able to test things nicely. I'm open to suggestions.